esp/scm/ws_config2.ecm

```diff
@@ -29,7 +29,7 @@ ESPstruct StatusMsgType
 ESPstruct StatusType
 {
     bool Error(false);    // true if a message exsits in status (below) with a message level of error or higher
-    ESParray<ESPstruct StatusMsgType, StatusMsg> Status;
+    ESParray<ESPstruct StatusMsgType, StatusMsg> Messages;
```

Comment:
this is ok, but typically ESParray<ESPstruct StatusMsgType, StatusMessage> StatusMessages;